src/mistral-client.ts

```diff
@@ -142,95 +131,27 @@ class MistralClient {
           }
           return await response.json();
         } else if (RETRY_STATUS_CODES.includes(response.status)) {
-          console.debug(
-            `Retrying request on response status: ${response.status}`,
-            `Response: ${await response.text()}`,
-            `Attempt: ${attempts + 1}`,
-          );
-          // eslint-disable-next-line max-len
-          await new Promise((resolve) =>
-            setTimeout(resolve, Math.pow(2, (attempts + 1)) * 500),
-          );
+          console.debug(`Retrying request on response status: ${response.status}`);
```

Comment:
You've dropped some of the text here:
"""
            `Response: ${await response.text()}`,
            `Attempt: ${attempts + 1}`,
"""